### PR TITLE
fundamentals material: Make links open in new tabs

### PR DESCRIPTION
**Pull request by gkunz**
In the older material, links still open in the current browser window which causes the user to navigate away from the PECA player. This is confusing and can break the learning experience.

### CODE REVIEW

**Review by tomas-ostling-lt — COMMENTED** [review]
I think this is a good change, so should be approved and functioality for all images should be confirmed in the Player. /tomas

File: Fundamentals-2-Ind.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Such strategy or policy might also be needed on product or product area level.
 
 ![Infrastructure](./img/Infrastructure.png)
 
-_Image by [Randall Munroe](https://xkcd.com/2347/)_
+_Image by [Randall Munroe](https://xkcd.com/2347/){:target="_blank"}_
```
Comment (YasinAhmed2) [inline Fundamentals-2-Ind.md:28]: Very good keeping the original page open while allowing readers to explore linked page outside.

File: Fundamentals-2-Ind.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Documentation to declare included free open source software components as either
 
 Suggested formats are being developed both for general consumption and for industry niches like automotive, health and telecom. There is a standard for the data format (SPDX) for this in place. ISO/IEC 5962:2021
 
-[Link to NTIA requirements on SBOMs >>](https://www.ntia.doc.gov/report/2021/minimum-elements-software-bill-materials-sbom)
+[Link to NTIA requirements on SBOMs >>](https://www.ntia.doc.gov/report/2021/minimum-elements-software-bill-materials-sbom){:target="_blank"}
```
Comment (YasinAhmed2) [inline Fundamentals-2-Ind.md:91]: Very good keeping the original page open while allowing readers to explore linked page outside.

**Review by YasinAhmed2 — APPROVED** [review]
I have not been part of the Fundamentals Coarse. I checked the PECA player. I agree the changes would make the learning experience much better by not moving away from the coarse material.

Very good keeping the original page open while allowing users to explore linked page outside.

I did not review the content otherwise. Ok, to merge from side.

**Pull request merged**